### PR TITLE
Update cacerts for debian 13 & ubuntu 24

### PR DESCRIPTION
**Pull request by steelhead31**
Add updates for new distros...

### CODE REVIEW

**Comment by steelhead31** [comment]
I'll need to republish the JDK8, JDK11 , JDK17 packages with a spec bump for the April release after this is merged.

**Review by gdams — APPROVED** [review]

**Review by sxa — APPROVED** [review]
Rubber stamping on the assumption you've checked this is everywhere the update is required but it superficially LGTM :-)

**Pull request merged**
